src/test/java/org/zendesk/client/v2/RealSmokeTest.java

```diff
@@ -1669,6 +1786,38 @@ private Ticket newTestTicket() {
         return ticket;
     }
 
+    /**
+     * Creates several new ticketImport (2 min, 5 max)
+     */
+    private TicketImport[] newTestTicketImports() {
+        final ArrayList<TicketImport> ticketImports = new ArrayList<>();
+        for (int i = 0; i < 2 + RANDOM.nextInt(3); i++) {
+            ticketImports.add(newTestTicketImport());
+        }
+        return ticketImports.toArray(new TicketImport[0]);
+    }
+
+    /**
+     * Creates a new ticketImport
+     */
+    private TicketImport newTestTicketImport() {
+        assumeThat("Must have a requester email", config.getProperty("requester.email"), notNullValue());
+        assumeThat("Must have a requester name", config.getProperty("requester.name"), notNullValue());
+        Date now = Calendar.getInstance().getTime();
```

Comment:
Is it ok to use now?

Reply:
What do you mean ? The name of the variable ?

Reply:
yes it's because it's an integration test